charts/datastore/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "datastore.fullname" . }}
+  labels:
+    product: {{ template "datastore.name" . }}
+    app: rookout
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      product: {{ template "datastore.name" . }}
+  template:
+    metadata:
+      labels:
+        product: {{ template "datastore.name" . }}
+        app: rookout
+    spec:
+      containers:
+        - name: {{ template "datastore.name" . }}
+          image: "{{ .Values.image.repository }}:{{ default "latest" .Values.image.tag }}"
+          env:
+            - name: ROOKOUT_DOP_SERVER_MODE
+              value: {{ .Values.datastore.serverMode | quote }}
+            - name: ROOKOUT_DOP_PUBLIC_KEY
+              value: {{ .Values.datastore.publicKey | quote }}
```

Comment:
You resolved the convo but this is still here? The chart shouldn't make the public key configurable

Reply:
oops, removed from values and forgot here